content/guides/hot-module-replacement.md

```diff
@@ -94,6 +88,13 @@ __index.js__
   }
 
   document.body.appendChild(component());
++
```

Comment:
oh I did mean that the new line was added, but if that's how it has been done on the other guides is fine 👍